tlair/src/main/scala/at/forsyte/apalache/tla/lir/Builder.scala

```diff
@@ -755,7 +755,8 @@ class Builder {
         ApalacheInternalOper.selectInSet.name -> ApalacheInternalOper.selectInSet,
         ApalacheInternalOper.storeInSet.name -> ApalacheInternalOper.storeInSet,
         ApalacheInternalOper.storeNotInSet.name -> ApalacheInternalOper.storeNotInSet,
-        ApalacheInternalOper.smtMap.name -> ApalacheInternalOper.smtMap,
+        ApalacheInternalOper.smtMap(TlaBoolOper.and).name -> ApalacheInternalOper.smtMap(TlaBoolOper.and),
+        ApalacheInternalOper.smtMap(TlaBoolOper.or).name -> ApalacheInternalOper.smtMap(TlaBoolOper.or),
```

Comment:
`.name` resolves to the same value in both cases so the bottom entry will override the top

Reply:
Good catch, thanks!